### ISSUE TITLE
Switch "policy -trigger" to "policy -event"

### ISSUE THREAD

**Issue by delize** [dan-snelson/Setup-Your-Mac#38]
> **Note:** While all requests are welcome, finding available cycles to custom-code a feature we won't use in our production environment is always challenging.

**Is your feature request related to a problem? Please describe.**
Future proof event/trigger call, as currently Jamf states that the 

```
# sudo jamf help policy

Usage:	 jamf policy [-event <event>] [-username <username>] [-id <policy_id>] [-forceNoRecon]

	-event		The event or trigger that the policy is associated with in the JSS. Historical synonyms include –trigger and –action.
				Note: Running policy without an event will default to the scheduled event.
				Other events include: login, logout, startup, networkStateChange, enrollmentComplete, along with custom events.

	-username		The username to check for policies for.

	-id			The ID of the policy to be executed. Used by Casper Remote and Self Service.

	-forceNoRecon		Prevents computers from submitting inventory update when a policy is configured to update inventory.
```

It would be good to update the script to start to use `-event`, in the case that Jamf decides to get rid of the historical use cases of the flag.

**Comment by dan-snelson** [comment]
Thanks, @delize

**Comment by dan-snelson** [comment]
@delize: This _should_ be addressed in [`1.10.0-rc26`](https://github.com/dan-snelson/Setup-Your-Mac/blob/development/Setup-Your-Mac-via-Dialog.bash).

**Comment by dan-snelson** [comment]
Thanks, @delize!
